tests/virtiofs/datavolume.go

```diff
@@ -335,5 +337,116 @@ var _ = Describe("[sig-storage] virtiofs", decorators.SigStorage, func() {
+
+			By("getting kubevirt client")
+			virtCli := kubevirt.Client()
+
+			By("creating a privileged pod")
+			podName := "info-pod"
+			debugPod := &k8sv1.Pod{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: podName,
+				},
+				Spec: k8sv1.PodSpec{
+					Containers: []k8sv1.Container{
+						{
+							Name:    "informer",
+							Image:   "busybox",
+							Command: []string{"/bin/sh", "-c", "sleep 3600"},
+							Stdin:   true,
+							TTY:     true,
+							SecurityContext: &k8sv1.SecurityContext{
+								Privileged: ptr.To(true),
+							},
+						},
+					},
+					HostNetwork: true,
+					HostPID:     true,
+					HostIPC:     true,
+					NodeName:    vmi.Status.NodeName,
+				},
+			}
+			pod, err := virtCli.CoreV1().Pods(testsuite.NamespacePrivileged).Create(context.TODO(), debugPod, metav1.CreateOptions{})
+			Expect(err).NotTo(HaveOccurred())
+
+			By("waiting until the pod is running")
+			Eventually(func() k8sv1.PodPhase {
+				pod, err := virtCli.CoreV1().Pods(testsuite.NamespacePrivileged).Get(context.TODO(), podName, metav1.GetOptions{})
+				Expect(err).NotTo(HaveOccurred())
+				return pod.Status.Phase
+			}, 30*time.Second, 1*time.Second).Should(Equal(k8sv1.PodRunning))
+
+			By("extracting process cgroup and NS info")
+			script := "export PID1=$(pidof virtiofs-placeholder) " +
+				"; grep ^0 /proc/$PID1/cgroup " +
+				"&& echo $(cd /proc/$PID1/ns && echo net pid ipc mnt cgroup uts | xargs -n1 readlink) " +
+				"&& export PID2=$(pidof virtiofsd) " +
+				"; grep ^0 /proc/$PID2/cgroup " +
+				"&& echo $(cd /proc/$PID2/ns && echo net pid ipc mnt cgroup uts | xargs -n1 readlink)" +
+				"&& awk -e '/^PPid:/ { print ENVIRON[\"PID1\"] == $2 }' /proc/$PID2/task/$PID2/status"
+
+			command := []string{"/bin/sh", "-c", script}
+
+			stdOut, stdErr, err := execute.ExecuteCommandOnPodWithResults(pod, pod.Spec.Containers[0].Name, command)
+			Expect(err).NotTo(HaveOccurred())
+			Expect(stdErr).To(BeEmpty())
+
+			By("comparing process cgroup and NS")
+			// we got:
+			// 		0: virtiofs-placeholder's cgroup
+			// 		1: virtiofs-placeholder's namespaces
+			// 		2: virtiofsd's cgroup
+			// 		3: virtiofsd's namespaces
+			//		4: "1" if the virtiofs-placeholder is the virtiofsd's parent process, "0" otherwise
+			out := strings.Split(stdOut, "\n")
+			Expect(out).To(HaveLen(6)) // strings.Split() adds an extra empty string
+			Expect(out[0]).Should(Equal(out[2]))
+			Expect(out[1]).Should(Equal(out[3]))
+			Expect(out[4]).Should(Equal("1"))
+
+			By("deleting privileged pod")
+			// Clean up the pod
+			err = virtCli.CoreV1().Pods(testsuite.NamespacePrivileged).Delete(context.TODO(), podName, metav1.DeleteOptions{})
+			Expect(err).NotTo(HaveOccurred())
+
+			By("deleting VMI")
+			err = virtClient.VirtualMachineInstance(vmi.Namespace).Delete(context.Background(), vmi.Name, metav1.DeleteOptions{})
+			Expect(err).ToNot(HaveOccurred())
+			libwait.WaitForVirtualMachineToDisappearWithTimeout(vmi, 120)
+		})
```

Comment:
Is the VMi deletion necessary? Usually, this is done automatically by the test cleanup